roles/deploy_user/tasks/main.yml

```diff
@@ -12,11 +12,37 @@
     home: "/home/{{ deploy_user }}"
     shell: "{{ deploy_user_shell }}"
 
-- name: deploy_user | add SSH keys
-  ansible.posix.authorized_key:
-    user: "{{ deploy_user }}"
-    key: "{{ item.key }}"
-  with_items: "{{ deploy_ssh_users }}"
+- name: deploy_user | get key content from github
```

Comment:
It would be nice to have a comment here, that this is utilized in the template.

Reply:
addressed in 5691a35087107bf8fe04c3b967b4e5bfaf537aec